bin/main/org/fundacionjala/at15/katas/README.md

```diff
@@ -0,0 +1,43 @@
+# Katas folder
+Welcome to katas folder for AT15!
```

Comment:
Please don't forget to remove `bin` folder

Reply:
I did it

Reply:
The bin folder was created again, delete it and push again, that should solve the problem